### DISCUSSION TITLE
大佬，怎么才能把脚本执行时机靠前

### DISCUSSION THREAD

**Discussion by zch233** [lisonge/vite-plugin-monkey#125] [Q&A]
目前我在 main.js 写的代码都会在页面加载之后才执行，如何才能在页面刚进入的时候就加载一些代码

![image](https://github.com/lisonge/vite-plugin-monkey/assets/30221824/fda3bdf5-ff45-4fbe-b43e-1c9da27f2db9)

![image](https://github.com/lisonge/vite-plugin-monkey/assets/30221824/d193a1ea-45d2-4fda-b9b5-50a0f181e1f6)

![image](https://github.com/lisonge/vite-plugin-monkey/assets/30221824/cc77994f-064a-4eab-8da7-9b493d1c7b65)

**Comment by lisonge** [comment]
设置 `config.userscript` 的属性
```txt
"run-at":'document-start'
```
